### ISSUE TITLE
0.108.0 Do not save integrated workflows

### ISSUE THREAD

**Issue by AllanDaemon** [n8n-io/n8n#1462]
**Describe the bug**
In the 0.108.0, then executing a sub-workflow with the `Execute Workflow` node, It doesn't save its execution anymore like before.

**To Reproduce**
Create a sub-workflow and a super-workflow that calls the former, them call the super-workflow.

`subflow`
```json
{
  "name": "subflow test",
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        250,
        300
      ]
    },
    {
      "parameters": {
        "values": {
          "string": [],
          "boolean": [
            {
              "name": "fromSubflow",
              "value": true
            }
          ]
        },
        "options": {}
      },
      "name": "Set",
      "type": "n8n-nodes-base.set",
      "typeVersion": 1,
      "position": [
        450,
        300
      ]
    }
  ],
  "connections": {
    "Start": {
      "main": [
        [
          {
            "node": "Set",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "active": false,
  "settings": {},
  "id": "1"
}
```

`superflow`
```json
{
  "name": "superflow test",
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        250,
        300
      ]
    },
    {
      "parameters": {
        "workflowId": "1"
      },
      "name": "Execute Workflow",
      "type": "n8n-nodes-base.executeWorkflow",
      "typeVersion": 1,
      "position": [
        550,
        300
      ]
    },
    {
      "parameters": {
        "values": {
          "boolean": [
            {
              "name": "fromSuper",
              "value": true
            }
          ]
        },
        "options": {}
      },
      "name": "Set",
      "type": "n8n-nodes-base.set",
      "typeVersion": 1,
      "position": [
        400,
        300
      ]
    }
  ],
  "connections": {
    "Start": {
      "main": [
        [
          {
            "node": "Set",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "Execute Workflow": {
      "main": [
        []
      ]
    },
    "Set": {
      "main": [
        [
          {
            "node": "Execute Workflow",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "active": true,
  "settings": {},
  "id": "2"
}
```

and call the webhook (or just execute it manually).

**Expected behavior**
Before, it showed like this:
![image](https://user-images.githubusercontent.com/607994/108337593-095bdf80-71b4-11eb-8865-3d0aaac66708.png)

Now, just the super workflow is saved (the one with mode `webhook`), no saved execution of with mode `integrated. It shows integrated executions from the past that were run in previous versions.

Also, setting `EXECUTIONS_DATA_SAVE_MANUAL_EXECUTIONS=true` and `EXECUTIONS_DATA_SAVE_ON_PROGRESS=true` did not helped.

**Environment:**
 - OS: Linux Fedora 33
 - n8n Version: `n8n@0.108.0`
 - Node.js Version: `v14.15.4`

**Comment by janober** [comment]
Just to update you. We are already working on a solution since yesterday. But turns out to be a little bit more complicated than anticipated. Will keep you updated.

**Comment by AllanDaemon** [comment]
> Just to update you. We are already working on a solution since yesterday. But turns out to be a little bit more complicated than anticipated. Will keep you updated.

Thanks. By now I just downgrading to 0.107.0 just do the work.

**Comment by AllanDaemon** [comment]
This was fixed by 6ee9501d1617295b98bac486073095d24760ad22 and released in 0.109.0 version.

Just a note:

Before, the sub-flow was saved before than the super-flow. Now, it's saving after, which is better and surely is what people would expect. But I just think that worth mentioning this in the changelog.
